packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-data-tile/OrganizationDataProtectionTile.spec.tsx

```diff
@@ -22,18 +22,29 @@ import {
   assertTextVisibility,
   getElementByTestId,
 } from '@ovh-ux/manager-core-test-utils';
+import userEvent from '@testing-library/user-event';
 import OrganizationDataProtectionTile from './OrganizationDataProtectionTile.component';
 import { labels } from '../../../test-utils';
 import {
   DATA_PROTECTION_BACKUP_LABEL,
   DATA_PROTECTION_RECOVERY_LABEL,
 } from '../../../pages/dashboard/organization/organizationDashboard.constants';
 import TEST_IDS from '../../../utils/testIds.constants';
+import { TRACKING } from '../../../tracking.constant';
```

Comment:
Be consistent in naming rules
```suggestion
import { TRACKING } from '../../../tracking.constants';
```